parser.py

```diff
@@ -640,16 +642,14 @@ def getInterventions(row):
 Main function to grab the WHO records for clinical trials.
 """
 
-def getWHOTrials(url, country_file, col_names, returnDF=False):
+def getWHOTrials(DATA_PATH, country_file, col_names, returnDF=False):
     today = date.today().strftime("%Y-%m-%d")
     # Natural Earth file to normalize country names.
     try:
         ctry_dict = pd.read_csv(country_file).set_index("name").to_dict(orient="index")
 
-        raw_req = Request(WHO_URL)
-        raw_req.add_header('User-Agent', 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:77.0) Gecko/20100101 Firefox/77.0')
-        raw_content = urlopen(raw_req)
-        raw = pd.read_csv(raw_content, dtype={"Date registration3": str})
+        raw_content = DATA_PATH+'COVID19-web.xlsx'
```

Comment:
@gtsueng can you change this to:
`raw_content = os.path.join(DATA_PATH, 'COVID19-web.xlsx'`

Reply:
requested change made